### PR TITLE
ODT - > PDF Numbered list #145

### PR DESCRIPTION
**Pull request by hpham**
Hi. 
please check

Greet Hanh

### CODE REVIEW

File: thirdparties-extension/org.odftoolkit.odfdom.converter.pdf/src/main/java/org/odftoolkit/odfdom/converter/pdf/internal/ElementVisitorForIText.java [hunk 1]
```diff
@@ -597,11 +605,15 @@ public void visit( TextListItemElement ele )
 
     @Override
     protected void visitImage( DrawImageElement ele, String href, byte[] imageStream )
-    {
-        // add image in the pdf.
-        Image imageObj = StylableImage.getImage( imageStream );
-        if ( imageObj != null )
-        {
+    {
```
Comment (angelozerr) [inline thirdparties-extension/org.odftoolkit.odfdom.converter.pdf/src/main/java/org/odftoolkit/odfdom/converter/pdf/internal/ElementVisitorForIText.java]: Please keep original indentation.

Comment (hpham) [inline thirdparties-extension/org.odftoolkit.odfdom.converter.pdf/src/main/java/org/odftoolkit/odfdom/converter/pdf/internal/ElementVisitorForIText.java]: Hi,

which eclipse formatter or intellij do you use?

**Comment by angelozerr** [comment]
Thank's @hpham !

**Comment by angelozerr** [comment]
@hpham build don't pass https://opensagres.ci.cloudbees.com/job/xdocreport/709/console

Could you see the problem please. Thanks

**Comment by hpham** [comment]
Hi, I do not understand the problem. the build is success. it seems the artifact can not be packaging or deploying. 
Greet Hanh

**Comment by angelozerr** [comment]
@pascalleclercq could you see the problem. Thanks!

**Pull request merged**